logcheck uses /run/lock by default so fix the path for systems without
/var/lock -> /run/lock symlink

diff --git a/Makefile b/Makefile
index fded3699..2a992473 100644
--- a/Makefile
+++ b/Makefile
@@ -13,7 +13,7 @@ install:
 	install -d $(DESTDIR)/var/lib/logcheck
 	install -d $(DESTDIR)/$(SBINDIR)
 	install -d $(DESTDIR)/$(BINDIR)
-	install -d $(DESTDIR)/var/lock/logcheck
+	install -d $(DESTDIR)/run/lock/logcheck
 	install -d $(DESTDIR)/$(SHAREDIR)
 
 	# Create directories for rules logcheck-database